content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -299,34 +323,25 @@ Also calling `kubeadm upgrade plan` and upgrading the CNI provider plugin is no
 <!--
 ### Upgrade kubelet and kubectl
 
--  Upgrade the kubelet and kubectl
+-  Upgrade the kubelet and kubectl:
 -->
 ### 升级 kubelet 和 kubectl
 
-- 升级 kubelet 和 kubectl
+- 升级 kubelet 和 kubectl：
 
   {{< tabs name="k8s_install_kubelet" >}}
   {{% tab name="Ubuntu、Debian 或 HypriotOS" %}}
-
-  <pre>
-  # 用最新的补丁版本替换 {{< skew currentVersion >}}.x-00 中的 x
-  apt-mark unhold kubelet kubectl && \
-  apt-get update && apt-get install -y kubelet={{< skew currentVersion >}}.x-00 kubectl={{< skew currentVersion >}}.x-00 && \
-  apt-mark hold kubelet kubectl
-  - 
-  # 从 apt-get 的 1.1 版本开始，你也可以使用下面的方法：
-  apt-get update && \
-  apt-get install -y --allow-change-held-packages kubelet={{< skew currentVersion >}}.x-00 kubectl={{< skew currentVersion >}}.x-00
-  </pre>
+      # 用最新的补丁版本替换 {{< skew currentVersion >}}.x-00 中的 x
+      apt-mark unhold kubelet kubectl && \
+      apt-get update && apt-get install -y kubelet={{< skew currentVersion >}}.x-00 kubectl={{< skew currentVersion >}}.x-00 && \
+      apt-mark hold kubelet kubectl
```

Comment:
please use explicit code snippet marker (i.e. three backtiqs) rather than indentation for code segments.

Reply:
We know it should be ` ```code block ``` `, but the [original doc](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md?plain=1#L177-L188) use this format instead.

Reply:
> please use explicit code snippet marker (i.e. three backtiqs) rather than indentation for code segments.

原文档用到了 {{< skew currentVersion >}} 来展示数据，文档中需要用到这个格式